src/components/BasicAuth/BasicAuthFlow/LandingContent.tsx

```diff
@@ -50,6 +55,14 @@ export function LandingContent({
         <br />
 
         <Stack spacing={4}>
+          {docsURL && (
+            <DocsURL
+              url={docsURL}
+              provider={providerName}
+              credentialName="basic auth credentials"
```

Comment:
Maybe just "credentials", instead of "basic auth credentials". Since the UI library is consumer facing, and they probably don't know basic auth means, nor do they need to know.